ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java

```diff
@@ -320,4 +322,12 @@ public void addProperty(String name, String value, String type) {
         select.selectOption(type);
         dialog.$(Element.PRIMARY_BUTTON).shouldBe(visible).click();
     }
+
+    public void switchToDatamapperIframe() {
+        switchTo().frame($(Element.ATLASMAP_IFRAME).shouldBe(visible).toWebElement());
+    }
+
+    public void switchBack() {
```

Comment:
I do not like the name of this method

Reply:
would be switchContextBack() better ?

Reply:
Yes and I would also add a comment why these methods exist and are necessary